index.html

```diff
@@ -272,6 +272,78 @@ <h3>Virtual Reality Gaming</h3>
+  by explicitly showing they do not have access to contents of their
+  users' calls. They are trusted to connect the right people to the
+  conference and to route the packets but they are not trusted to access
+  the audio and video media or text in the call.</p>
+
+<p>Solutions to this problem fall into two major categories: one where
+  the JavaScript comes from a source trusted to see the media contents,
+  and one where it does not:</p>
+
+<h4>Untrusted JavaScript Cloud Conferencing</h4>
+
+<p>There are many cases where a system such as WebEx is trusted to
+  connect the members of a conference but has no need to access the
+  contents of the conference. This is true of the majority of
+  conferencing systems on the web today.  Just to highlight the
+  scope of this requirement, there are more minutes of WebRTC
+  that are used in conferences where the servers have no need
+  to access the contents (e.g. where audio is forwarded rather than
+  mixed) than any other use of WebRTC audio by orders of magnitude.
+  This is one of the primary use case for WebRTC audio and accounts
+  for billions of minutes per month of potential use of WebRTC.
+</p>
+
+<p>In this use case, the JavaScript comes from the operator of the
+  conference bridge. The isolated media features of WebRTC prevent
+  the JavaScript from accessing the media and the identity features
+  are used to provide a user interface that allows the user to know
+  it connected to the correct conference. The goal is for the end
+  users to be able to see the contents, but the web service that
+  provides the JS and the media switching bridges and Selective
+  Forwarding Units (SFUs) cannot access the contents (audio, video,
+  text). Some metadata about the audio, such as the power
+  levels of the audio media, are encrypted end-to-end and are 
+  therefore not accessible to the media server.
+</p>
+
+<p>A possible solution this problem is the browser to negotiate end to
+  end encryption keys which are not revealed to the JavaScript.
+</p>
+
+<h4>Trusted JS</h4>
+
+<p>In some highly controlled situations such as military uses,
+  the JavaScript can come from a trusted source while the servers
+  that distribute the packets are not trusted to access the
+  contents in cleartext. The servers are trusted to forward
+  packets as appropriate and not launch denial of service attacks
+  on the conferences.</p>
+
+<p>In this case, the JavaScript running the WebRTC application comes
+  from a trusted source and is allowed to access the contents in
+  cleartext. These use cases tend to be relatively small deployments
+  as they require the JavaScript come from a trusted source, but they
+  use a media server from an untrusted source. The contents need to be
+  encrypted with keys that are not known to the operator of the servers,
+  servers, then sent to the conference server. It is acceptable for
+  the power levels of the audio content to be revealed to the server
```

Comment:
Isn't this true for both cases?

Reply:
Not sure why the server is allowed access to the power levels in one sub-use-case and not the other one.

Reply:
I think the requirement should be that it's possible to inform the server about the power levels without revealing any other information about the content of the media (and this would apply in both use cases).
I don't think it makes sense to make an explicit non-requirement saying "it's OK if you always have to reveal the power level to the server", which is what the "it is acceptable" text currently seems to say.

Reply:
Added the same text (suggested by Harald) to both use cases.